app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
@vbudhram - You are right, that makes sense. We would only want to log a `signin.success` if `this.isSignIn()` is true. I was just tracing through this part of [views/confirm.js](https://github.com/mozilla/fxa-content-server/blob/153292b1778fc0976622995d5694f39f8a72bc3d/app/scripts/views/confirm.js#L107) and realized even logging a `signin.success` metric there misses out on a couple types of users:

1. Unverified users who sign in from about:accounts. When embedded in about:accounts, FxA never starts the verification poll because Firefox is taking care of it. Since the poll is never started, the poll will never complete, and the event will never log.
2. Unverified users from the firstrun page who close the FxA tab before verifying their email.

IIUC, this is exactly why some activity events are emitted from the auth server instead of content server client code.

I think these holes are why some events are emit as activity events from the auth server rather than from from the content server. Since most of our users come from the firstrun flow, this might not be *too* bad of a problem, and we are still closer to the "real" count.